docs/src/integrate/patina_dxe_core_requirements.md

```diff
@@ -165,6 +165,42 @@ Resource Descriptor HOB v2 describing it.
 > Platforms must create Resource Descriptor HOB v2s for all memory resources including MMIO and reserved memory with
 > a valid cacheability attribute set.
 
+#### 2.2.1 Resource Descriptor Hob v2 Guidance
+
+The guidance is that Resource Descriptor Hobs need to define the system's memory map. Some have read this
+as only needing to describe the memory that is in use, but others have read it as needing to describe the
+entire memory map. The Resource Descriptor Hobs should describe the system memory map, as it stands, at
+the end of PEI. This includes caching attributes, so that patina is able to consume the data and provide
+enhanced memory portections from the start of DXE.
+
+##### PEI Video Example
+
+By the start of DXE, most system memory should be configured as Write Back (EFI_MEMORY_WB), and most Memory Mapped I/O
+should be configured as uncachable (EFI_MEMORY_UC). There are exceptions to this, based upon platform level decisions.
+For example, if the pre DXE phase configured a video device and displayed something on the screen, then the memory
+region associated with the frame buffer, for performance, should be reflected for that region of memory.
```

Comment:
I would flesh this out a bit more if you are giving an example, say the frame buffer being marked as write combining (or whatever it should be)

Reply:
The frame buffer is a good example to walk through its caching transitions throughout boot. Usually, it will be WC for perf either to display a pre-DXE boot graphic or in DXE for display and then moved to UC before handing off to the OS. Describing the interfaces a flow like that should use would probably be insightful guidance.